parsers/saarland.py

```diff
@@ -0,0 +1,199 @@
+from urllib.request import *
+
+import json
+import datetime
+
+from pyopenmensa.feed import LazyBuilder
+
+from utils import Parser, Source
+
+
+API_VERSION = "1"
+# please don't copy this key, contact felix@fefrei.de to get your own instead
+API_KEY = "A1RT5M56" + "ibsGaGNZBJQg"
+APP_VERSION = "1"
+SUPPORTED_LANGUAGES = ["de", "en"]
+
+URL_BASE = (
+    'https://mensaar.de/api/%s/%s/%s/%s/'
+    % (API_VERSION, API_KEY, APP_VERSION, SUPPORTED_LANGUAGES[0])
+)
+URL_BASE_DATA = "getBaseData"
+URL_MENU = "getMenu/"
+
+UTC_DATE_STRING = "%Y-%m-%dT%H:%M:%S.%fZ"
```

Comment:
Yes. As per the spec, the `date` field on a *day* object contains *a date representing the corresponding day at midnight, UTC*.

Reply:
Maybe you could add a comment linking to the spec?

Reply:
The specification isn't public (so far).

Reply:
Oh, ok. Maybe just a comment stating the reliance on the spec? Just so that when it changes and someone reads the code, there is an explicit dependency stated, so that they'll catch it more likely.

Something like `Mensaar.de always returns an UTC date` or similar? Or what do you think of such a comment?

Reply:
I agree, it's added.